packages/transactions/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -2,36 +2,17 @@ import React, { FC } from 'react';
 
 import { useNavigate, useParams } from 'react-router';
 
-import {
-  request,
-  Transaction,
-  useQueryClient,
-} from '@openmsupply-client/common';
+import { Transaction, useQueryClient } from '@openmsupply-client/common';
 
-import { getMutation, getDetailQuery } from '../../api';
+import { detailQueryFn, updateTransactionFn } from '../../api';
```

Comment:
Extracted the functions in the `ListView` and `DetailView` out to the `api` file which contains all the code for the detail+list queries, delete mutation and upsert mutations as I reused the upsert mutation in the `ListView`

Reply:
Think these will pretty much all go into custom hooks at some point